packages/forms/backend/sirius-components-collaborative-forms/src/main/java/org/eclipse/sirius/components/collaborative/forms/dto/SelectListItemInput.java

```diff
@@ -0,0 +1,85 @@
+/*******************************************************************************
+ * Copyright (c) 2022 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.components.collaborative.forms.dto;
+
+import java.text.MessageFormat;
+import java.util.Objects;
+import java.util.UUID;
+
+import org.eclipse.sirius.components.collaborative.forms.api.IFormInput;
+import org.eclipse.sirius.components.forms.ClickEventKind;
+
+/**
+ * The input object for the list item selection mutation.
+ *
+ * @author fbarbin
+ */
+public final class SelectListItemInput implements IFormInput {
+    private UUID id;
+
+    private String representationId;
+
+    private String editingContextId;
+
+    private String listId;
+
+    private String listItemId;
+
+    private ClickEventKind onClickEventKind;
```

Comment:
minor nitpick but why not `clickEventKind`?

Reply:
agree :-)